### PR TITLE
show total upload and download

### PR DESCRIPTION
**Pull request by clkao**
Closes https://github.com/karissa/hyperdrive-ui/issues/34

### CODE REVIEW

**Comment by karissa** [comment]
seems to work great!

File: src/js/components/hyperdrive-size/index.js [hunk 1]
```diff
@@ -30,8 +30,10 @@ HyperdriveSize.prototype._updateSize = function (state) {
     var s = state.archiveReducer
     if (s.archive && s.archive.content && s.archive.content.bytes) {
       this._size = prettyBytes(s.archive.content.bytes)
+      this.$el.style.display = 'block'
     } else {
       this._size = 0
+      this.$el.style.display = 'none'
```
Comment (karissa) [inline src/js/components/hyperdrive-size/index.js]: not entirely sure we have to do this right now b/c we still need to design this thing anyway. cc @Kriesse

**Comment by karissa** [comment]
screen shot 2016-07-07 at 11 18 28 am

File: src/js/components/hyperdrive-size/index.js (hunk 1 above)
Comment (clkao) [inline src/js/components/hyperdrive-size/index.js]: the reducer in the size component wasn't run for the downloading client, and it'll be 0 all the time and weird.  i suggest we hide that for now and rework size styles along with the reducer fixes.

Comment (laurengarcia) [inline src/js/components/hyperdrive-size/index.js]: this can be easily fixed in another PR, can you just add a `TODO: fix` comment here instead? also, if we do any dom manipulations re: style display or block, they should always go in component _render() method, otherwise it can be confusing.

Comment (clkao) [inline src/js/components/hyperdrive-size/index.js]: ok splited. had to tweak the css as well

**Comment by karissa** [comment]
nice job @clkao

**Comment by karissa** [comment]
nice job @clkao!
